meson_options.txt

```diff
@@ -0,0 +1 @@
+option('skip_tests', type : 'boolean', value : false)
```

Comment:
I think it would be better to have a "positive" option (like `build_tests`) that defaults to true, as it may be a bit more intuitive to use. We might even consider *disabling* them by default since they are mostly useless for the end user.

Reply:
Done